awx/main/conf.py

```diff
@@ -277,6 +277,16 @@ def _load_default_license_from_file():
     placeholder={'HTTP_PROXY': 'myproxy.local:8080'},
 )
 
+register(
+    'AWX_ROLES_ENABLED',
+    field_class=fields.BooleanField,
+    default=True,
+    label=_('Enable Role Download'),
+    help_text=_('Allows roles to be dynamically downlaoded from Galaxy based on a requirements.yml file.'),
```

Comment:
this should not be 'from Galaxy'... the ansible-galaxy command will read from whatever is specified as the 'src' regardless of Galaxy metadata.

Reply:
(repeat as necessary elsewhere)